src/components/app.js

```diff
@@ -7,13 +7,14 @@ import NavHeader from './common/nav-header';
 import Products from './products';
 import Product from './product';
 import Register from './user/register';
+import Login from './user/login';
 import Home from './home';
 import { ROUTES } from '../constants';
 import '../stylesheets/app.scss';
 
 const history = createHistory();
 const {
-  home, register, products, product,
+  home, register, products, product, logIn,
```

Comment:
Login (noun) is one word. So you can change this to all lowercase: "login"

Reply:
Ah okay. My bad, I thought register was also a verb and I wanted to follow the same pattern so I went with `Log In` as in like the actual verb to log in. But now I see the register is also a noun.... I will update it.